stable/mongodb-replicaset/templates/mongodb-init-configmap.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
     app: {{ template "mongodb-replicaset.name" . }}
     chart: {{ template "mongodb-replicaset.chart" . }}
     heritage: {{ .Release.Service }}
-    release: {{ .Release.Name }}
+    release: {{ .Release.Name | trunc 63 | trimSuffix "-"  }}
```

Comment:
I don't like this, especially doing that redundantly across the chart. I'm not aware of any other chart that does this. Why would anyone chose release names this long?

Reply:
the release name is being populated from ci/cd and I was quite happy to truncate my end before going into the chart, but I noticed the chart was trying to be proactive about the character limit in the templates, so wanted to help